src/android/notification/Options.java

```diff
@@ -242,7 +242,7 @@ public int getLedColor() {
         String hex = options.optString("led", null);
 
         if (hex == null) {
-            return NotificationCompat.DEFAULT_LIGHTS;
+            return 0;
```

Comment:
`NotificationCompat.DEFAULT_LIGHTS` is supposed to be a simple flag for NotificationCompat.setDefaults method, not a RGB color